content/docs/2.5/scalers/rabbitmq-queue.md

```diff
@@ -39,7 +39,8 @@ triggers:
 - `queueLength`: DEPRECATED! Use `mode: QueueLength` and `value: ##` instead. Target value for queue length passed to the scaler. Example: if one pod can handle 10 messages, set the queue length target to 10. If the actual number of messages in the queue is 30, the scaler scales to 3 pods. Default is 20 unless `publishRate` is specified, in which case `queueLength` is disabled for this trigger.
 - `useRegex`: In case of `http` protocol, this parameter allows to use regex (in `queueName` parameter) to select queue instead of full name, the valid values are: `"true"` and `"false"`.  Optional.
 - `operation`: Operation that will be applied to compute the number of messages in case of `useRegex` enabled. Either `sum` (default),`max`, or `avg`. Optional.
-- `metricName` - an optional name to assign to the metric. If not set KEDA will generate a name based on the queue name. If using more than one trigger it is required that all metricNames be unique.
+- `metricName`: an optional name to assign to the metric. If not set KEDA will generate a name based on the queue name. If using more than one trigger it is required that all metricNames be unique.
+- `timeout`: Timeout **for this specific trigger** in case of `http` protocol. This value will override the value defined in `KEDA_HTTP_DEFAULT_TIMEOUT`. Optional.
```

Comment:
@JorTurFer does the first sentence mean to say that the timeout only applies for this specific trigger, only if it's the `http` protocol? If so, here's a suggestion:

```suggestion
- `timeout`: Timeout **for this specific trigger**. This value will override the value defined in `KEDA_HTTP_DEFAULT_TIMEOUT`. Optional. Only apples to hosts that use the `http` protocol
```

Reply:
Nice suggestion! Thanks for do it :)

Reply:
No problem :)